utils/node_local_test.py

```diff
@@ -908,6 +998,68 @@ def run_tests(dfuse):
     symlink_dest = 'missing_dest'
     os.symlink(symlink_dest, symlink_name)
     assert symlink_dest == os.readlink(symlink_name)
+    readdir_test(dfuse, 10)
+
+def dfuse_wrapper(server, conf):
+    """Start a dfuse instance, do something then tear it down"""
+
+    pools = get_pool_list()
+    while len(pools) < 1:
+        pools = make_pool(server)
+
+    pool = pools[0]
+
+    container = create_cont(conf, pool, ctype='POSIX')[0]
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'ctype' in function call